packages/unlock-js/src/unlockService.ts

```diff
@@ -208,16 +208,13 @@ export default class UnlockService {
   async getHookContract({
     network,
     address,
-    signer,
```

Comment:
@kalidiagne You should not need a signer to instantiate a contract. The provider is enough.